### DISCUSSION TITLE
Multiplayer exclusive mods?

### DISCUSSION THREAD

**Discussion by CenTdemeern1** [ppy/osu#18138] [Ideas] [answered]
Are multiplayer exclusive mods a thing, or planned to be a thing?
Mods that interact with other players sound like an interesting idea.
For example: cursor confusion, (a “fun” category mod) where you can see all the other player’s cursors, and your own cursor blends in with the rest

**Comment by peppy** [comment, marked as answer]
We have plans to add new multiplayer modes. They will likely not be implemented as mods, but a different system.

> For example: cursor confusion, (a “fun” category mod) where you can see all the other player’s cursors, and your own cursor blends in with the rest

This is night impossible due to latency. Already been suggested 100s of times.
